requirements/default.txt

```diff
@@ -1 +1 @@
-celery>=4.0,<5.0
+celery>=4.2.1,<5.0
```

Comment:
Why this change? Do the tests fail without it?

Reply:
For some reason I had the impression that `apply()` was not present in earlier versions, but (on double checking) I was wrong.  I'll remove this change.